dispatcher/backend/src/common/schemas/offliners/youtube.py

```diff
@@ -139,146 +213,105 @@ class Meta:
+            "description": "Custom banner image. Will be resized to 1060x175px",
         }
     )
-    playlists_title = String(
+
+    main_color = HexColor(
         metadata={
-            "label": "Playlists title",
-            "description": "Custom title format for individual playlist ZIM",
+            "label": "Main Color",
+            "description": "Custom color. Hex/HTML syntax (#DEDEDE). "
+            "Default to main color of profile image.",
         },
-        data_key="playlists-title",
+        data_key="main-color",
     )
 
-    description = String(
-        metadata={"label": "Description", "description": "Description for ZIM"},
-        validate=validate_zim_description,
-    )
-    playlists_description = String(
+    secondary_color = HexColor(
         metadata={
-            "label": "Playlists description",
-            "description": "Custom description format for individual playlist ZIM",
+            "label": "Secondary Color",
+            "description": "Custom secondary color. Hex/HTML syntax (#DEDEDE). "
+            "Default to secondary color of profile image.",
         },
-        data_key="playlists-description",
+        data_key="secondary-color",
     )
 
-    creator = String(
-        metadata={
-            "label": "Content Creator",
-            "description": "Name of content creator. Defaults to Channel name "
-            "or “Youtue Channels”",
-        }
-    )
-    tags = String(
-        metadata={
-            "label": "ZIM Tags",
-            "description": "List of Tags for the ZIM file. "
-            "_videos:yes added automatically",
-        }
+    debug = fields.Boolean(
+        truthy=[True],
+        falsy=[False],
+        metadata={"label": "Debug", "description": "Enable verbose output"},
     )
 
     metadata_from = String(
         metadata={
             "label": "Metadata JSON",
-            "description": "File path or URL to a JSON file holding custom metadata "
-            "for individual playlists",
+            "description": "Expert flag: File path or URL to a JSON file holding "
+            "custom metadata for individual playlists",
         },
         data_key="metadata-from",
     )
 
-    profile = fields.Url(
+    api_key = String(
         metadata={
-            "label": "Profile Image",
-            "description": "Custom profile image. Squared. "
-            "Will be resized to 100x100px",
-        }
+            "label": "API Key",
+            "description": "Expert flag: Youtube API Token",
+            "secret": True,
+        },
+        data_key="api-key",
+        required=True,
     )
-    banner = fields.Url(
+
+    optimization_cache = fields.Url(
         metadata={
-            "label": "Banner Image",
-            "description": "Custom banner image. Will be resized to 1060x175px",
-        }
+            "label": "Optimization Cache URL",
+            "description": "Expert Flag: S3 Storage URL including credentials and "
```

Comment:
Not an expert field. Content people hopefully knows what this is and why it should always be set and according to the scraper. It's not edited often though and is a technical flag but should be checked.

Reply:
Moved up front, it is important indeed you are right